public/scripts/main.js

```diff
@@ -192,6 +192,27 @@ npc.isSeeing(player);
 layer.add(npc.render);
 
 
+function httpGet(theUrl) {
+  const xmlHttp = new XMLHttpRequest();
+  xmlHttp.open('GET', theUrl, false); // false for synchronous request
+  xmlHttp.send(null);
+  return xmlHttp.responseText;
+}
+
+// Draw a face
+const text = httpGet('/cool');
+const face = new Konva.Text({
```

Comment:
Could you add the face obj as an object within the Character class? Then update its x/y from within the move() method so it moves in the canvas

Reply:
You can init it in the character class constructor, too

Reply:
You can get the width and height from a Konva.Text obj. Could use that to center or make sure the size of the character obj contains the ascii face

Reply:
Yup, that's the plan, once I put it in the character class, this PR should be ready to merge